tools/render-test/render-d3d11.cpp

```diff
@@ -40,34 +40,28 @@ using namespace Slang;
 
 namespace renderer_test {
 
-//static char const* vertexEntryPointName    = "vertexMain";
-//static char const* fragmentEntryPointName  = "fragmentMain";
-
-//static char const* vertexProfileName   = "vs_4_0";
-//static char const* fragmentProfileName = "ps_4_0";
-
 class D3D11Renderer : public Renderer, public ShaderCompiler
 {
 public:
     // Renderer    implementation
     virtual SlangResult initialize(void* inWindowHandle) override;
-    virtual void setClearColor(float const* color) override;
+    virtual void setClearColor(const float color[4]) override;
     virtual void clearFrame() override;
     virtual void presentFrame() override;
     virtual SlangResult captureScreenShot(char const* outputPath) override;
-    virtual void serializeOutput(BindingState* state, const char * fileName) override;
-    virtual Buffer* createBuffer(BufferDesc const& desc) override;
-    virtual InputLayout* createInputLayout(InputElementDesc const* inputElements, UInt inputElementCount) override;
+    virtual void serializeOutput(BindingState* state, const char* fileName) override;
+    virtual Buffer* createBuffer(const BufferDesc& desc) override;
```

Comment:
We don't have any kind of official policy on this, but I tend to write `T const&` and `T const*` just because it properly generalizes the the more complex cases (e.g., `char const* const*` vs. `char* const*` vs. `char const**`).

I'm fine with either convention, but I want us to be careful about not doing cleanup work just for the sake of doing cleanup, especially on stuff where there is no hard-and-fast rule. My general approach is to not edit a line of code unless I really have to as part of my current work, but then if I *do* have to touch the line I'll go ahead and make whatever small cleanups seem appropriate at the time.

In this case it doesn't matter because nobody else is touching these files, but as you dig into more parts of the compiler, it will be important for us to keep our diffs small and focused to help avoid merge pain for others on the team.